lib/ansible/modules/windows/win_pester.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2018, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = r'''
+---
+module: win_pester
+short_description: Run Pester tests on Windows hosts
+version_added: "2.6"
+description:
+  - Run Pester tests on Windows hosts.
+  - Test files have to be available on the remote host.
+requirements:
+  - The Pester module should be present on the remote host.
```

Comment:
You only need to list the component name (and optionally minimum version). The documentation will generate the supporting text.